Packs/AnsibleMicrosoftWindows/Integrations/AnsibleMicrosoftWindows/AnsibleMicrosoftWindows.yml

```diff
@@ -0,0 +1,7030 @@
+category: IT Services
```

Comment:
Looking at output examples in the README there seems to be a large number of outputs not mapped here.
Is that on purpose? It looks more like a bug in the integration generation.